plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.scss

```diff
@@ -6,9 +6,19 @@
 	justify-content: space-between;
 }
 
-a.woocommerce-marketplace__refresh-subscriptions {
+.woocommerce-marketplace__refresh-subscriptions {
 	text-decoration: none;
 	color: #007cba;
+
+	&.is-busy {
+		@keyframes spin {
+			0% { transform: rotate(0deg); }
+			100% { transform: rotate(360deg); }
+		  }
+		.woocommerce-marketplace__refresh-subscriptions__icon {
+			animation: spin 2s linear infinite;
+		}
+	}
```

Comment:
Can we please add this on line 24 to address one of Poli's comments in 18757-gh-Automattic/woocommerce.com?

> Refresh button:
>  Is that the right spacing between the icon and text? Is this something coming from Woo core? There should be spacing between the icon and text, 4px.

```scss
.woocommerce-marketplace__refresh-subscriptions__icon {
	margin-right: $grid-unit-05;
}
```